lib/tasks/data.rake

```diff
@@ -75,4 +76,137 @@ namespace :aleftng do
       end
     end
   end
+
+  def find_and_replace_in_text(find, replace, text)
+    #puts "Search and replace #{find.inspect} => #{replace.inspect}"
+    text.gsub!(find, replace)
+    return text
+  end
+
+  # TODO - Výnimka - iný typ otázky ako tieto základné
+  def find_question_type(text)
+    if text == "single-choice"
```

Comment:
Ked by to uz malo byt cez podmienky, tak treba pouzit switch, nie elsif spagetu. Ale na toto je uplne idealny Hash. V zmysle (nie nutne doslovne):

IMPORTED_QUESTION_TYPES = {
  single-choice: 'SingleChoiceQuestion',
  // ...
}

type = IMPORTED_QUESTION_TYPE[str.to_sym]